home.html

```diff
@@ -0,0 +1,155 @@
+            <h3 id="date">2022.10.26(WED) ~ 28(FRI)</h3>
+            <p id="date-location">@ICC Central Business District, Abuja</p>
+    </section>
+
+    <section class="program-info" id="program">
+        <h2 id="main-program">Main Program</h1>
+        <div class="program-bar"></div>
+        <div class="program">
+        <div class="program-list">
+            <div class="program-icon-wrapper">
+                <img class="program-icon" src="img/program_icon_01.png" alt="drum sticks icon">
+            </div>
+            <h4 class="program-topic">Drum Chops Presentation</h4>
+            <p class="program-text">Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                Molestiae qui odit fugiat nemo maiores earum, sunt saepe veniam sequi eum.</p>
+        </div>
+        <div class="program-list">
+            <div class="program-icon-wrapper">
+                <img class="program-icon" src="img/music-media.jpg" alt="drum sticks icon">
+            </div>
+            <h4 class="program-topic">Advertising & Media</h4>
+            <p class="program-text">Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                Molestiae qui odit fugiat nemo maiores earum, sunt saepe veniam sequi eum.</p>
+        </div>
+        <div class="program-list">
+            <div class="program-icon-wrapper">
+                <img class="program-icon" src="img/program_icon_03.png" alt="drum sticks icon">
+            </div>
+            <h4 class="program-topic">Building Connection</h4>
+            <p class="program-text">Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                Molestiae qui odit fugiat nemo maiores earum, sunt saepe veniam sequi eum.</p>
+        </div>
+        <div class="program-list">
+            <div class="program-icon-wrapper">
+                <img class="program-icon" src="img/program_icon_02.png" alt="drum sticks icon">
+            </div>
+            <h4 class="program-topic">Drum Brands Presentaion</h4>
+            <p class="program-text">Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                Molestiae qui odit fugiat nemo maiores earum, sunt saepe veniam sequi eum.</p>
+        </div>
+        <div class="program-list">
+            <div class="program-icon-wrapper">
+                <img class="program-icon" src="img/program_icon_01.png" alt="drum sticks icon">
+            </div>
+            <h4 class="program-topic">Drum Brands</h4>
+            <p class="program-text">Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                Molestiae qui odit fugiat nemo maiores earum, sunt saepe veniam sequi eum.</p>
+        </div>
+        </div>
+        <a class="see-more" href="#">SEE THE WHOLE PROGRAMS</a>
+    </section>
+
+    <section class="speaker-section">
+        <h2 class="speaker-heading">Featured Speakers</h2>
+        <div class="program-bar"></div>
+        <div class="speaker-container" id="speakers" >
+        </div>
+    </section>
```

Comment:
- [ ] It would be nice if you could give the images the same size so as to match the given design, you could use the `height` property to achieve this

Your design
![image](https://user-images.githubusercontent.com/81932478/196775449-d1936c01-45aa-4ab3-a19f-4eac7cd17100.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/196775612-7f25b2f1-fd4b-4f8b-8b50-27314a2004c4.png)